nautobot/extras/templates/extras/role_retrieve.html

```diff
@@ -0,0 +1,96 @@
+                <td>Slug</td>
+                <td>
+                    {{ object.slug }}
+                </td>
+            </tr>
+            <tr>
+                <td>Weight</td>
+                <td>
+                    {{ object.weight|placeholder }}
+                </td>
+            </tr>
+            <tr>
+                <td>Content Type(s)</td>
+                <td>
+                    <ul class="list-unstyled">
+                        {% for ct in content_types %}
+                            <li>{{ ct }}</li>
+                        {% endfor %}
+                    </ul>
+                </td>
+            </tr>
+            <tr>
+                <td>Color</td>
+                <td>
+                    <span class="label color-block" style="background-color: #{{ object.color }}">&nbsp;</span>
+                </td>
+            </tr>
+            <tr>
+                <td>Description</td>
+                <td>
+                    {{ object.description|placeholder }}
+                </td>
+            </tr>
+        </table>
+    </div>
+{% endblock content_left_page %}
+
+{% block content_full_width_page %}
+    <div class="panel panel-default">
+         <div class="panel-heading">
+             <strong>IPAddresses</strong>
+         </div>
+         {% include 'inc/table.html' with table=ipaddress_table %}
+         {% if perms.ipam.add_ipaddress %}
+             <div class="panel-footer text-right noprint">
+                 <a href="{% url 'ipam:ipaddress_add' %}?role={{ object.pk }}" class="btn btn-xs btn-primary">
+                     <span class="mdi mdi-plus-thick" aria-hidden="true"></span> Add IPAddress
+                 </a>
+             </div>
+         {% endif %}
+     </div>
+     <div class="panel panel-default">
+         <div class="panel-heading">
+             <strong>Prefixes</strong>
+         </div>
+         {% include 'inc/table.html' with table=prefix_table %}
+         {% if perms.ipam.add_prefix %}
+             <div class="panel-footer text-right noprint">
+                 <a href="{% url 'ipam:prefix_add' %}?role={{ object.pk }}" class="btn btn-xs btn-primary">
+                     <span class="mdi mdi-plus-thick" aria-hidden="true"></span> Add prefix
+                 </a>
+             </div>
+         {% endif %}
+     </div>
+     <div class="panel panel-default">
+         <div class="panel-heading">
+             <strong>VLANs</strong>
+         </div>
+         {% include 'inc/table.html' with table=vlan_table %}
+         {% if perms.ipam.add_vlan %}
+             <div class="panel-footer text-right noprint">
+                 <a href="{% url 'ipam:vlan_add' %}?role={{ object.pk }}" class="btn btn-xs btn-primary">
+                     <span class="mdi mdi-plus-thick" aria-hidden="true"></span> Add vlan
+                 </a>
+             </div>
+         {% endif %}
+     </div>
+     {% include 'inc/paginator.html' with paginator=vlan_table.paginator page=vlan_table.page %}
```

Comment:
Seems off to me that we include paginator.html for the VLAN table but not for the IPAddress and Prefix tables?

Also, any reason we don't also list Devices, VirtualMachines, and Racks using this role?